mslib/mscolab/mscolab.py

```diff
@@ -67,24 +71,25 @@ def confirm_action(confirmation_prompt):
             print("Invalid input! Please select an option between y or n")
 
 
-def handle_db_init():
-    from mslib.mscolab.models import db
-    from mslib.mscolab.server import APP
-    create_files()
-    with APP.app_context():
-        db.create_all()
-    print("Database initialised successfully!")
-
-
 def handle_db_reset(verbose=True):
-    from mslib.mscolab.models import db
-    from mslib.mscolab.server import APP
-    if os.path.exists(mscolab_settings.DATA_DIR):
+    if mscolab_settings.SQLALCHEMY_DB_URI.startswith("sqlite:///") and (
```

Comment:
can we still reset other dbases?

Reply:
Yes, the issue with SQLite is that that the database file is inside DATA_DIR, but deleting it breaks the ORM. This simply is not an issue with other databases, since they do not reside in DATA_DIR as a file.